src/middleware/auth/admin/adminAuthDataOnlyMiddleware.ts

```diff
@@ -12,18 +9,14 @@ const adminAuthDataOnlyMiddleware = async (req: Request, res: Response, next: Ne
 
   if (adminAccessToken == null) return res.sendStatus(401);
 
-  const TOKEN = process.env.ADMIN_ACCESS_TOKEN_SECRET;
-
   let admin;
 
-  if (TOKEN === undefined) console.log("No secret access token in env");
-  else
-    try {
-      admin = jwt.verify(adminAccessToken, TOKEN);
-    } catch (err) {
-      console.log(err);
-      return res.sendStatus(403);
-    }
+  try {
+    admin = jwt.verify(adminAccessToken, process.env.JWT_SECRET_ADMIN);
+  } catch (err) {
+    console.log(err);
+    return res.sendStatus(403);
```

Comment:
Better to send an error to express (`next(error)`). You can use this to generate nice errors: https://www.npmjs.com/package/http-errors